models/topics.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../db/connection')
+
+
+exports.getAllTopics = (req,res) => {
+    return db.query(`SELECT * FROM topics`)
+    .then(resp => resp)
```

Comment:
If this `then` block simply returns what it is given - it can actually be removed completely